app/store/local-adoption.ts

```
@@ -0,0 +1,169 @@
+			storage.value = validTokens;
+
+			// next, clear local map data
+			const tokensToRemove: string[] = [];
+
+			this.adoptableProbes.forEach((fetchedAt, key) => {
+				if (now - fetchedAt >= 1000 * 61) {
+					tokensToRemove.push(key);
+				}
+			});
+
+			tokensToRemove.forEach(t => this.adoptableProbes.delete(t));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)